src/encoded/static/components/item.js

```diff
@@ -6,11 +6,14 @@ import { FetchedData, Param } from './fetched';
 import { JSONSchemaForm } from './form';
 import * as globals from './globals';
 import { AlternateAccession } from './objectutils';
+import AllSchemasPage from './schema';
 
 
 const Fallback = (props, reactContext) => {
     const context = props.context;
-    const title = typeof context.title === 'string' ? context.title : url.parse(reactContext.location_href).path;
+    const path = url.parse(reactContext.location_href).path;
+    const title = typeof context.title === 'string' ? context.title : (path === '/profiles/' ? 'Schemas' : path);
```

Comment:
I'm not a big fan of adding special cases to the `Fallback` component like this; it feels like sidestepping the view lookup system, and it puts code for a very specific case into a component that is meant to be very generic. Is there a reason we can't add a `@type` to the response from `/profiles/`? If so, maybe it would be better to make globals.contentViews.fallback check a mapping of path to `@type` before using this `Fallback` component?

Reply:
yeah I don't like this either, should be fixed on the backend.

Reply:
Would this be a snovault change?

Reply:
yeah I think so.

Reply:
OK. I’ll look into that.

Reply:
develop/snovault/src/snovault/schema_views.py:

```
@view_config(route_name='schemas', request_method='GET',
             decorator=etag_app_version_effective_principals)
def schemas(context, request):
    types = request.registry[TYPES]
    schemas = {}
    for type_info in types.by_item_type.values():
        name = type_info.name
        schemas[name] = _annotated_schema(type_info, request)

    schemas['_subtypes'] = subtypes = {}
    schemas['@type'] = ['JSONSchemas'];
    for name, type_info in types.abstract.items():
        subtypes[name] = type_info.subtypes

    return schemas
```

Results in:

```
...
        "changelog": "/profiles/changelogs/antibody_characterization.md",
        "@type": [
            "JSONSchema"
        ]
    },
    "@type": [
        "JSONSchemas"
    ],
    "TreatmentConcentrationSeries": {
        "title": "Treatment concentration series",
        "description": "Schema for submitting metadata for a treatment concentration series.",
        "id": "/profiles/treatment_concentration_series.json",
...
```